client/jira.go

```diff
@@ -226,6 +235,21 @@ func AssignIssue(issueId, username string) error {
 	return err
 }
 
+func GetTransitions(issueId string) (TransitionsResult, error) {
+	result := TransitionsResult{Transitions: []Transition{}}
```

Comment:
nitpicky: feel free to ignore this but we can omit `Transitions: []Transitions` since our struct will be initialised with zero-values, the Transitions field will automatically be initialised to an empty slice, if that makes sense